### DISCUSSION TITLE
How do I know if there are conflicting files?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1066] [Q&A] [answered]
What would happen if two of the folders that form the pool have a file in the same path? And how do I find this files so that I can change the name?

**Comment by trapexit** [comment]
Things work exactly as described in the documentation. Every function has a policy and the policy dictates what is used.

**Comment by ghost** [comment, marked as answer]
`find dir1 dir2 -printf '%P\n' | sort | uniq -d`

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs#file--directory-xattrs
